noir/noir-repo/tooling/nargo_cli/src/cli/debug_cmd.rs

```diff
@@ -187,7 +187,11 @@ fn run_async(
             }
 
             if let Some(witness_name) = witness_name {
-                let witness_path = save_witness_to_dir(solved_witness, witness_name, target_dir)?;
+                let witness_path = save_witness_to_dir(
```

Comment:
Seems like the debugger should just return a `WitnessStack` instead.

Reply:
I could make that switch. I held off on that as it required threading the new logic throughout the `debugger` when it does not support multiple ACIR calls yet.